exasol_script_languages_developer_sandbox/lib/cf_stack.py

```diff
@@ -17,19 +15,23 @@ class CloudformationStack:
     and when exiting the stack will be destroyed.
     """
 
-    def __init__(self, aws_access: AwsAccess, ec2_key_name: str, user_name: str):
+    def __init__(self, aws_access: AwsAccess, ec2_key_name: str, user_name: str, stack_prefix: Optional[str]):
         self._aws_access = aws_access
         self._stack_name = None
         self._ec2_key_name = ec2_key_name
         self._user_name = user_name
+        if stack_prefix is None:
+            self._stack_prefix = "EC2-SLC-DEV-SANDBOX-"
```

Comment:
you should be able to put the default directly to the parameter. This is only an issue if the default has a mutable type, strings shouldn't be mutable.

Reply:
as I pass through the prefix in function `run_lifecycle_for_ec2`, I would have to add a check there, too. Which makes it even more complicated

Reply:
ok, you are right